### DISCUSSION TITLE
DBeaver under Citrix

### DISCUSSION THREAD

**Discussion by GeriZ** [dbeaver/dbeaver#8563] [Q&A] [answered]
#### System information: 
- Windows 2016
- DBeaver 7.03

#### Describe your question:
Is DBeaver officially supported under Citrix?

**Comment by serge-rider** [comment, marked as answer]
Yes, it is.
We have many customers who run dbeaver on Citrix.

**Comment by GeriZ** [comment]
Thanks for your information!

**Reply by 786abhi786** [reply]
which version of DBEAVER you installed on citrix ?

**Reply by 786abhi786** [reply]
Is there any specific settings for the functionality of DBeaver in Citrix ?

**Reply by GeriZ** [reply]
7.2.1. Nothing special

**Comment by GeriZ** [comment]
7.2.1 No special settings
